src/Config/Registrar.php

```diff
@@ -16,9 +17,10 @@ public static function Filters(): array
     {
         return [
             'aliases' => [
-                'session' => SessionAuth::class,
-                'tokens'  => TokenAuth::class,
-                'chain'   => ChainAuth::class,
+                'session'    => SessionAuth::class,
+                'tokens'     => TokenAuth::class,
+                'chain'      => ChainAuth::class,
+                'auth-limit' => AuthRates::class,
```

Comment:
Why is the classname `AuthRates`, not `AuthLimit`?

Reply:
Better question - why is it `auth-limit` when the class is named `AuthRates` :)  I started it as auth limit then changed my mind and missed that. Fixed.

Reply:
Done.